### PR TITLE
DAOS-4049 control: replace go-bytesize with go-humanize

### PR DESCRIPTION
**Pull request by tanabarr**
go-humanize can represent IEC (GiB) and SI (GB) storage units.
Internally store raw byte counts and translate to the relevant units
(IEC for memory, SI for storage) at user interfaces. Accept both
unit types in pool create commandline parameters.

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

**Comment by tanabarr** [comment]
needed as dependency for the following fix https://github.com/daos-stack/daos/pull/1944

File: src/control/server/storage/types.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2019 Intel Corporation.
+// (C) Copyright 2019-20w20 Intel Corporation.
```
Comment (kjacque) [inline src/control/server/storage/types.go]: oops

Comment (tanabarr) [reply src/control/server/storage/types.go]: this seems to have slipped in with https://github.com/daos-stack/daos/pull/1898 , moved fix to https://github.com/daos-stack/daos/pull/1944/commits/ba17c2c83406964c03ce6c0aec59f39cd9b5cc93 which will get applied to both master and release branches.

**Comment by tanabarr** [comment]
https://jira.hpdd.intel.com/browse/DAOS-3945 intermittent failure https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-1969/3/tests

**Pull request merged**